vm/src/vm/vm_ops.rs

```diff
@@ -151,7 +151,7 @@ impl VirtualMachine {
     /// Calling scheme used for binary operations:
     ///
     /// Order operations are tried until either a valid result or error:
-    ///   b.rop(b,a)[*], a.op(a,b), b.rop(b,a)
+    ///   b.rop(a,b)[*], a.op(a,b), b.rop(a,b)
```

Comment:
`rop` is a reversed version of `op`. So (b,a) is intended.

Reply:
Done. Had changed the comment looking at https://github.com/python/cpython/blob/main/Objects/abstract.c#L920

Reply:
Oh, thank you! we'd better to follow it later.